src/entity/src/user.js

```diff
@@ -718,7 +725,7 @@ export default class User {
     // Check if this is the active user
     if (this.isActive() === false) {
       return Promise.reject(
-        new KinveyError('This user must be the active user to register from real time.')
+        new KinveyError('This user must be the active user in order to register for real time.')
```

Comment:
Umm this part of the `User` class I had left to discuss when we discuss or start merging the collection and user-to-user logic. I didn't want to break the collection before we decided what we're doing. Cause I think the register logic shouldn't even be in the `User` class, I think it belongs to the `LiveService` class.